ros2/diagnostics/README.md

```diff
@@ -0,0 +1,12 @@
+# Diagnostics basic use
```

Comment:
This doesn't seem to have anything to do with the rust tutorial, did you mean to include it here?

Reply:
I pulled from the main branch where the diagnostics tutorial had been already created. 
I did it to sync my branch with main and avoid merge conflicts.